src/components/SoonInTheCinema.vue

```diff
@@ -9,35 +9,50 @@
                     <router-link to="/movies" class="movies__text movies__text--red">See All</router-link>
                 </div>
                 <div class="article__wrapper">
-                    <!-- <MovieCard/> -->
-                    
+
+                    <article class="article">
+                        <h3 class="article__header">Predator</h3>
+                        <BaseTag text="1h 30min">{{text}}</BaseTag>
+                        <img src="@/assets/predator.png" alt="Predator" class="article__img">
+                        <BaseTag text="Action" class="tag--red"> {{text}}</BaseTag>
+                    </article>
+
                     <article class="article">
                         <h3 class="article__header">Commando</h3>
+                        <BaseTag text="1h30min">{{text}}</BaseTag>
```

Comment:
I can't see you are using [slots](https://v2.vuejs.org/v2/guide/components-slots.html) inside `BaseTag` so the content (`{{ text }}`) is not used 🤔

Reply:
btw. `text` seems to be always empty

Reply:
just to sum up: you should probably decide what do you want to achieve:
1. Usage of `BaseTag` as it is right now:
```html
<BaseTag text="some text" />
```
2. Usage of `BaseTag` with a default slot defined:
```html
<BaseTag>some text</BaseTag>
```
if you want option (2) to be working you need to change a little bit `BaseTag` component:
it needs to have a `<slot></slot>` in the place where you want children to be rendered